docs/source/advanced/targets.rst

```diff
@@ -140,3 +140,7 @@ in ``ProgramData/Microsoft/Windows/Hyper-V/data.vmcx`` that tells us where all o
 Then we can then use these paths and tell ``dissect.target`` to load another target from there. Reading all of these
```

Comment:
Wouldn't you want to make this text a bit more highover and move this in-depth info to the sub-page?